debian/rules

```diff
@@ -1,5 +1,6 @@
 #!/usr/bin/make -f  
 %:  
+	sed -i "s/Architecture: .*/Architecture: $(CONFIGURED_ARCH)/g" debian/control
```

Comment:
Not like this, Change in debian/control, Archiecture : any, that would be suffice.

Reply:
Done